src/devices/SensorHub/README.md

```diff
@@ -0,0 +1,10 @@
+﻿# SensorHub
+
+## Summary
+SensorHub for reading Temperature, Pressure, Illuminance and Humidity. I2C can be used to communicate with the device
```

Comment:
In Markdown, you have 1 empty after and before each title (except level 1 title, only after).

Reply:
Missing . at the end of the sentence

Reply:
believe this to be correct now